services/self-service/src/test/java/com/epam/dlab/backendapi/resources/ProjectResourceTest.java

```diff
@@ -181,4 +278,16 @@ private ProjectDTO returnProjectDTO(CreateProjectDTO createProjectDTO) {
     private ProjectActionFormDTO getProjectActionDTO() {
         return new ProjectActionFormDTO(PROJECT_NAME, Collections.singletonList("https://localhost:8083/"));
     }
+
+    private UpdateProjectDTO prepareUpdateProjectDTO() {
+        return new UpdateProjectDTO(PROJECT_NAME, Collections.emptySet(), Collections.emptySet(), true);
+    }
+
+    private ProjectActionFormDTO prepareProjectActionFormDTO(){
+        return new ProjectActionFormDTO(PROJECT_NAME, Collections.singletonList("https://localhost:8083/"));
+    }
+
+    private List<UpdateProjectBudgetDTO> prepareUpdateProjectBudgetDTOs() {
+        return Collections.singletonList(new UpdateProjectBudgetDTO(PROJECT_NAME, 123, false));
```

Comment:
Use **Boolean.FALSE** instead of just **false**

Reply:
Variable is changed